Sources/PublishCLICore/CLIError.swift

```diff
@@ -8,7 +8,7 @@ internal enum CLIError: Error {
     case newProjectFolderNotEmpty
     case notASwiftPackage
     case failedToResolveSwiftPackageName
-    case outputfolderNotFound
+    case outputFolderNotFound
```

Comment:
Looks like you have renamed the enum to proper camel-case but not replaced various uses of it in the code. You can always use Xcode's Refactor->Rename to avoid issues like this in future. At this point the project won't build with this change.

Reply:
thanks for the tip!